packages/response-ops/rule_form/src/rule_actions/rule_actions_item.tsx

```diff
@@ -692,7 +691,7 @@ export const RuleActionsItem = (props: RuleActionsItemProps) => {
       }
       buttonContentClassName="eui-fullWidth"
       buttonContent={
-        <EuiPanel color="subdued" paddingSize="m">
+        <EuiPanel color={subdued} paddingSize="m">
```

Comment:
Since we had to replace `useEuiBackgroundColor('subdued'), the closes color I found was `euiTheme.colors.lightestShade`, but if I hadn't also replaced the default subdued panel color, it would have looked something like this: 
<img width="1322" alt="Screenshot 2024-12-17 at 10 33 54" src="https://github.com/user-attachments/assets/2e280a49-4d3d-4cb6-9f46-82b51a5a5c0c" />

But still, it doesn't seem the best approach, since the colors supported by EuiPanel are the following: "transparent", "accent", "primary", "success", "warning", "danger", "subdued", "plain". I need to find a workaround.

Reply:
I see. Since you're adding a background color to the Accordion, would it work to use `<EuiPanel color="transparent" />` ?